mobile/src/screens/login/LoginScreen.tsx

```diff
@@ -26,37 +26,54 @@ export const LoginScreen: React.FC = () => {
-        style={[{marginTop: insets.top}, styles.backButtonContainer]}
-        onPress={() => navigation.goBack()}
-      >
-        <Image style={styles.backButton} source={require('../../../assets/images/Arrow_left.png')}/>
-      </Pressable>
-
-      <View style={styles.logoContainer}>
-        <Image source={require('../../../assets/images/logo-white.png')} style={styles.logo} />
-      </View>
-
-      <View style={styles.container}>
-        <TextInput style={[styles.input, TextStyles.caption3]} value={email} onChangeText={setEmail} placeholder={i18n.t('email')} placeholderTextColor={Colors.gray} textContentType="emailAddress" />
-        <TextInput style={[styles.input, TextStyles.caption3]} value={password} onChangeText={setPassword} placeholder={i18n.t('password')} placeholderTextColor={Colors.gray} secureTextEntry />
-
-        <TouchableOpacity style={styles.rememberMeContainer} onPress={() => setRememberMe(!rememberMe)}>
-          <Checkbox value={rememberMe} onChange={setRememberMe} inverted />
-          <Text style={styles.rememberMeText}>{i18n.t('rememberMe')}</Text>
-        </TouchableOpacity>
-
-        <View style={styles.signInContainer}>
-          <LargeButton text={i18n.t('signIn')} onPress={login} border />
+
+    <KeyboardAvoidingView
+      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+      style={{ flex: 1, height: '100%' }}
+    >
+
+      <ScrollView style={styles.background} keyboardShouldPersistTaps='handled' contentContainerStyle={styles.backgroundChildren}>
+        <Pressable
+          style={[{marginTop: insets.top}, styles.backButtonContainer]}
+          onPress={() => navigation.goBack()}
+        >
+          <Image style={styles.backButton} source={require('../../../assets/images/Arrow_left.png')}/>
+        </Pressable>
+
+        <View style={styles.logoContainer}>
+          <Image source={require('../../../assets/images/logo-white.png')} style={styles.logo} />
+        </View>
+
+        <View style={styles.container}>
+          <TextInput style={[styles.input, TextStyles.caption3]} value={email} onChangeText={setEmail} placeholder={i18n.t('email')} placeholderTextColor={Colors.gray} textContentType="emailAddress" />
+          <TextInput style={[styles.input, TextStyles.caption3]} value={password} onChangeText={setPassword} placeholder={i18n.t('password')} placeholderTextColor={Colors.gray} secureTextEntry />
+
+          <View style={styles.textContainer}>
+              
+            <TouchableOpacity style={styles.rememberMeContainer} onPress={() => setRememberMe(!rememberMe)}>
+              <Checkbox value={rememberMe} onChange={setRememberMe} inverted />
+              <Text style={styles.rememberMeText}>{i18n.t('rememberMe')}</Text>
+            </TouchableOpacity>
+
+            <TouchableOpacity style={styles.forgotPswdContainer} onPress={() => navigation.navigate("ForgotPswd")}>
+              <Text style={styles.forgotPasswordText}>{i18n.t('forgotPassword')} </Text>
```

Comment:
Maybe change the double quotes to single quotes (also in a couple of other places). I believe David is working on the linter though, so probably not a big issue.